backend/app/main.py

```
@@ -27,5 +27,17 @@ def create_task(payload: TaskCreate, db: Session = Depends(get_db)):
     db.refresh(task)
     return task
 
+@app.delete("/task/{task_id}",status_code=204)
+def delete_task(task_id=int,db:Session=Depends(get_db)):
```

Ruff (0.15.6)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)